### ISSUE TITLE
Failure to deploy SP Lists

### ISSUE THREAD

**Issue by Humpeklumpen** [OfficeDev/microsoft-teams-apps-requestateam#122]
First of all, thanks! I needed this app package!

Second, I get an error when I run the script;
### REQUESTS SPO SITE CONFIGURATION ###
Configuring SharePoint site...
Applying provisioning template...
Applied template
Error occured while configuring the SharePoint site: Cannot find an overload for "Load" and the argument count: "1".

And the SPO site is created, but not the lists. 
Any ideas?
Also complains of the AzureADpreview, but that wont install on my computer.. Everything else runs

**Comment by Humpeklumpen** [comment]
My bad, AzureADPreview wouldnt install with PS Ise x86..

**Comment by alexc-MSFT** [comment]
@Humpeklumpen can you post any more details as to the initial error please? Screenshots etc.

Also what version of PnP do you have as there is a known issue at the moment with the latest version.

**Comment by Humpeklumpen** [comment]
@alexc-MSFT - I fixed it. I start all my apps from using the windows key on my keyboard and then type what I want and hit Enter, so Windows PowerShell ISE x86 opened up, not my usual Powershell. And ISE x86 dont work with AzureADpreview. 
Once I realized, and changed, everything is ok.

Well, except the role assignment.
![az](https://user-images.githubusercontent.com/8370646/90509054-09bde400-e159-11ea-962d-0e20ee5af2cf.PNG)

Norwegian: Inndatastrengen hadde ugyldig format = "Input data string was invalid format"
